menu/menu.yaml

```
@@ -381,11 +381,11 @@
       Title: Microsoft Generic Host
     - Url: nservicebus/hosting/web-application
       Title: Web Application
-    - Url: nservicebus/hosting/azure-functions
-      Title: Azure Functions
+    - Url: nservicebus/hosting/azure-functions-service-bus
+      Title: Azure Functions for Service Bus
```

ok, sorry for nitpicking, I just saw that the docspage uses "Azure Functions with Azure Service Bus" title, so maybe the alternative is this:

```suggestion
      Title: Azure Functions with Service Bus
```

I'll let you decide. PR is already approved, good with either option.